test/snapshots/seam-blueprint.test.ts.md

```diff
@@ -10,6 +10,170 @@ Generated by [AVA](https://avajs.dev).
 
     {
       name: 'Seam Connect',
-      resources: {},
-      routes: [],
+      resources: {
+        acs_system: {
+          properties: [
+            {
+              deprecationMessage: '',
+              description: '',
+              isDeprecated: false,
+              name: 'acs_system_id',
+              type: 'string',
+            },
+            {
+              deprecationMessage: '',
+              description: '',
+              isDeprecated: false,
+              name: 'can_add_acs_users_to_acs_access_groups',
+              type: 'string',
+            },
+            {
+              deprecationMessage: '',
+              description: '',
+              isDeprecated: false,
+              name: 'can_automate_enrollment',
+              type: 'string',
+            },
+            {
+              deprecationMessage: '',
+              description: '',
+              isDeprecated: false,
+              name: 'can_create_acs_access_groups',
+              type: 'string',
+            },
+            {
+              deprecationMessage: '',
+              description: '',
+              isDeprecated: false,
+              name: 'can_remove_acs_users_from_acs_access_groups',
+              type: 'string',
+            },
+            {
+              deprecationMessage: '',
+              description: '',
+              isDeprecated: false,
+              name: 'connected_account_ids',
+              type: 'list',
+            },
+            {
+              deprecationMessage: '',
+              description: '',
+              isDeprecated: false,
+              name: 'created_at',
+              type: 'string',
+            },
+            {
+              deprecationMessage: '',
+              description: '',
+              isDeprecated: false,
+              name: 'errors',
+              type: 'list',
+            },
+            {
+              deprecationMessage: '',
+              description: '',
+              isDeprecated: false,
+              name: 'external_type',
+              type: 'string',
```

Comment:
This is supposed to be an enum

Reply:
That's a weird case since the type (which is the case for the switch statement) is string. Will have to pay attention to such cases once we generate the whole thing